impl/client/src/test/javascript/pentaho/visual/base/view.spec.js

```diff
@@ -819,7 +819,7 @@ define([
         // _updateSize is still updating
 
         // Change the view's selection
-        view.selectionFilter = null;
+        view.selectionFilter = {_: "or"};
```

Comment:
Why the change? Setting to `null` doesn't trigger the change detection?

Reply:
Setting to null triggers obtaining a new default value. However the default value semantics now has changed. Before we'd always clone the default value. Now, if that semantics is desired, a default value function must be supplied that creates a new instance each time.
The selectionFilter default value wasn't changed — as it is an instance of an immutable class — setting to `null` no longer yields a new value and, as such, does not trigger an on change anymore.